webpages/login.php

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en" >
+    <head>
+        <meta charset="UTF-8">
+        <title>Sign In</title>
+        
+        <link rel="stylesheet" href="../css/signup.css">
+    </head>
+    <body>
+         <?php
+            $email = $password = "";
+            $emailerr = $passerr = "";
+            function prepare($data){
+            
+              $data = trim($data);
+              $data = stripslashes($data);
+               $data = htmlspecialchars($data);
```

Comment:
indent, be consistent as to where you put the closing or the opening braces, with the functions as well as the if statements.